content/en/docs/Reference/file-based-catalogs.md

```diff
@@ -304,6 +304,48 @@ properties:
     kind: Bar
 ```
 
+#### `olm.deprecations`
+
+Operator authors can use the file-based catalog (FBC) deprecation schema to provide relevant information for support and upgrades for users running operators. The deprecation schema follows the Meta FBC Schema conventions. 
+
+Deprecation for packages/bundles/channels is enabled through the olm.deprecations schema. This schema consists of a list of scoped references to catalog content, with a descriptive message string that provides more information to the user. The deprecation schema is an optional feature.
+
+A valid depreciation schema meets the following criteria:
+- There must be only one schema per package
+- The message must be a non-zero length
+- The package name must exist in the bundle
+  
+The deprecation feature does not consider overlapping deprecation (package vs channel vs bundle).
+
+|              | `olm.package`                   | `olm.channel`        | `olm.bundle`          |
+|--------------|---------------------------------|----------------------|-----------------------|
+| Scope        | Entire Package                  | Single Channel       | Single Bundle Version |
+| Requirements | Must not be an associate `name` | `name`  is mandatory | `name`  is mandatory  |
```

Comment:
"Must not be an associate `name`" - what does that mean? 

Maybe a bit more clear would be something like:
```suggestion
| Requirements | `name` must be empty | `name`  is mandatory | `name`  is mandatory  |
```

And then either within that block _or_ in a separate piece of text we can explain that when referencing an `olm.package` in a deprecation the `name` is inherited from the `package` field (that is present for all FBC objects as it is part of the meta schema)